### PR TITLE
F3

### PR DESCRIPTION
**Pull request by CramBL**
Added feature with 2 buttons: AddTimeButton & SubtractTimeButton.

Can add/remove time from timer during cooking.

### CODE REVIEW

File: Microwave.App/Program.cs [hunk 1]
```diff
@@ -22,11 +24,11 @@ class Program
 
             Light light = new Light(output);
 
-            Microwave.Classes.Boundary.Timer timer = new Timer();
+            Microwave.Classes.Boundary.Timer timer = new Microwave.Classes.Boundary.Timer();
```
Comment (AUmav) [inline Microwave.App/Program.cs:28]: Wow, ingen forvirring med Microsoft Timer her !!!!

File: Microwave.Classes/Controllers/CookController.cs [hunk 2]
```diff
@@ -35,7 +35,7 @@ public class CookController : ICookController
 
             timer.Expired += new EventHandler(OnTimerExpired);
             timer.TimerTick += new EventHandler(OnTimerTick);
-
+
```
Comment (AUmav) [inline Microwave.Classes/Controllers/CookController.cs:42]: Sick programming dude!

**Comment by AUmav** [comment]
Det ser da meget fint ud!

File: Microwave.App/Program.cs [hunk 3]
```diff
@@ -22,11 +24,11 @@ class Program
...
 
             CookController cooker = new CookController(timer, display, powerTube);
 
-            UserInterface ui = new UserInterface(powerButton, timeButton, startCancelButton, door, display, light, cooker);
+            UserInterface ui = new UserInterface(powerButton, timeButton, startCancelButton, AddTimeButton, subtractTimeButton, door, display, light, cooker);
```
Comment (metthorna) [inline Microwave.App/Program.cs:34]: Smart

**Review by metthorna — APPROVED** [review]
This looks great! Good work

**Review by AUmav — APPROVED** [review]
Great

**Pull request merged**